script/production/mainnet/OwnershipTransferCore.s.sol

```diff
@@ -31,7 +31,7 @@ contract OwnershipTransferCore is ScriptUtils {
             FactoryGovernor(coreAddresses.eVaultFactoryGovernor).DEFAULT_ADMIN_ROLE(), EVAULT_FACTORY_GOVERNOR_ADMIN
         );
         FactoryGovernor(coreAddresses.eVaultFactoryGovernor).grantRole(
-            FactoryGovernor(coreAddresses.eVaultFactoryGovernor).GUARDIAN_ROLE(), EVAULT_FACTORY_GOVERNOR_ADMIN
+            FactoryGovernor(coreAddresses.eVaultFactoryGovernor).PAUSE_GUARDIAN_ROLE(), EVAULT_FACTORY_GOVERNOR_ADMIN
```

Comment:
Should it also be granted unpause powers?

Reply:
yep, good catch!